### ISSUE TITLE
`reflex build` with placeholder email doesn't update `reflex.yaml`

### ISSUE THREAD

**Issue by mcbanderson** [reflexivesecurity/reflex-cli#226]
Version: v1.2.2

When running `reflex build` with the placeholder contact email, you are prompted if you want to update the email address. If you select yes and then enter a new email address, the entered email address is used for the built terraform files but `reflex.yaml` is not updated.

**Comment by joerk2313** [comment]
Would the file Function work?

Example:
> file("${path.module}/hello.txt")
Hello World

https://www.terraform.io/docs/configuration/functions/file.html

**Comment by mcbanderson** [comment]
No, the file function only reads the contents of a file. And since that's a Terraform function we wouldn't be able to utilize it from the CLI anyways.

The relevant code is here: https://github.com/reflexivesecurity/reflex-cli/blob/master/reflex_cli/commands/command_build.py#L54-L77

After the configuration is updated on line 77 we'd then need to write that configuration to the configuration file, similar to what we do with `reflex update`.
